src/Schema/Tables/AllergySymptom.sql

```diff
@@ -0,0 +1,9 @@
+﻿CREATE TABLE [dbo].[AllergySymptom]
+(
+	[AllergySymptomId] INT NOT NULL PRIMARY KEY IDENTITY, 
+    [PatientAllergyId] INT NOT NULL, 
+    [SymptomId] INT NOT NULL, 
+    [IsDeleted] BIT NOT NULL, 
+    CONSTRAINT [FK_AllergySymptom_To_PatientAllergy] FOREIGN KEY (PatientAllergyId) REFERENCES PatientAllergy(PatientAllergyId), 
```

Comment:
[FK_AllergySymptom_To_PatientAllergy] - without [] and naming as in Anton`s example:
FK_Table1_Table2